data_summaries/COIG-Kun.json

```
@@ -0,0 +1,47 @@
+{
+    "coig-kun": {
+        "Unique Dataset Identifier": "coig-kun",
+        "Dataset Name": "COIG-Kun",
+        "Paper Title": "Kun: Answer Polishment Saves Your Time for Using Intruction Backtranslation on Self-Alignment",
+        "Dataset URL": "https://github.com/Zheng0428/COIG-Kun",
+        "GitHub URL": "https://github.com/Zheng0428/COIG-Kun",
+        "Hugging Face URL": "https://huggingface.co/datasets/m-a-p/COIG-Kun",
+        "Papers with Code URL": "",
+        "ArXiv URL": "",
+        "Semantic Scholar Corpus ID": "",
+        "Collection": "COIG-Kun",
+        "Collection URL": "https://huggingface.co/datasets/m-a-p/COIG-Kun",
+        "Languages": [
+            "Chinese"
+        ],
+        "Task Categories": [
+            "Question Answering"
+        ],
+        "Text Sources": [],
+        "Model Generated": ["m-a-p/Kun-LabelModel", "01-ai/Yi-34B-Chat"],
+        "Format": [
+            "Zero-shot"
+        ],
+        "Human Annotation": "No",
+        "Derived from Datasets": [
+            "skypile",
+            "wanjuan",
+            "wudao"
+        ],
+        "Creators": [
+            "01.ai",
+            "Multimodal Art Projection (M-A-P)"
+        ],
+        "Licenses": [
+            {
+                "License": "Apache License 2.0",
+                "License URL": "https://github.com/Zheng0428/COIG-Kun/blob/master/LICENSE"
```

```suggestion
                "License URL": "https://www.apache.org/licenses/LICENSE-2.0"
```

I think we do it like this no @shayne-longpre ?

---

I think we're fine with where it actually points to the original license, as its easier for a future reviewer/verifier to find where we made the determination.